fire/core.py

```diff
@@ -287,15 +292,15 @@ def _DisplayError(component_trace):
     print('INFO: Showing help with the command {cmd}.\n'.format(
         cmd=pipes.quote(command)), file=sys.stderr)
     help_text = helptext.HelpText(result, trace=component_trace,
-                                  verbose=component_trace.verbose)
+                                  verbose=component_trace.verbose, help_sequence=help_sequence)
     output.append(help_text)
     Display(output, out=sys.stderr)
   else:
     print(formatting.Error('ERROR: ')
           + component_trace.elements[-1].ErrorAsStr(),
           file=sys.stderr)
     error_text = helptext.UsageText(result, trace=component_trace,
-                                    verbose=component_trace.verbose)
+                                    verbose=component_trace.verbose, help_sequence=help_sequence)
```

Comment:
```suggestion
                                    verbose=component_trace.verbose)
```

wrong addition of parameter in UsageText() function